### PR TITLE
Update main readme

### PR DESCRIPTION
**Pull request by lhoestq**
The main readme was still focused on text datasets - I extended it by mentioning that we also support image and audio datasets

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -197,45 +204,3 @@ If you want to cite our 🤗 Datasets [paper](https://arxiv.org/abs/2109.02846)
     primaryClass={cs.CL},
 }
 ```
-```bibtex
-@software{quentin_lhoest_2021_5639822,
```
Comment (lhoestq) [inline README.md:201]: I removed this @albertvillanova since researchers can just cite the paper, WDYT ?

**Comment by lhoestq** [comment]
What do you think @albertvillanova ?

**Review by albertvillanova — APPROVED** [review]
Thanks for the extension; much better now.

File: README.md (hunk 1 above)
Comment (lhoestq) [reply README.md:201]: I added a link to Zenodo with the list of all the released DOIs sorted by version

Comment (albertvillanova) [reply README.md:201]: Good idea! we should give both options for citation: either the library, or a specific version of our software package.

File: README.md [hunk 2]
```diff
@@ -147,7 +153,8 @@ Similar to TensorFlow Datasets, 🤗 Datasets is a utility library that download
 If you're a dataset owner and wish to update any part of it (description, citation, etc.), or do not want your dataset to be included in this library, please get in touch through a [GitHub issue](https://github.com/huggingface/datasets/issues/new). Thanks for your contribution to the ML community!
 
 ## BibTeX
-If you want to cite our 🤗 Datasets [paper](https://arxiv.org/abs/2109.02846) and library, you can use these:
+
+If you want to cite our 🤗 Datasets [paper](https://arxiv.org/abs/2109.02846) and library, you can use this:
```
Comment (albertvillanova) [inline README.md:157]: Some rephrasing, but feel free to ignore it:
```suggestion
If you want to cite our 🤗 Datasets library, you can use our [paper](https://arxiv.org/abs/2109.02846):
```

File: README.md [hunk 3]
```diff
@@ -197,45 +204,5 @@ If you want to cite our 🤗 Datasets [paper](https://arxiv.org/abs/2109.02846)
...
-  month        = nov,
-  year         = 2021,
-  publisher    = {Zenodo},
-  version      = {1.15.1},
-  doi          = {10.5281/zenodo.5639822},
-  url          = {https://doi.org/10.5281/zenodo.5639822}
-}
-```
+
+You can also cite the library using the Zenodo DOI (one DOI per version of the library), see the full list [here](https://zenodo.org/search?q=conceptrecid:%224817768%22&sort=-version&all_versions=True).
```
Comment (albertvillanova) [inline README.md:208]: Some rephrasing, mentioning "reproducibility" requirements, but feel free to ignore it as well.
```suggestion
If you need to cite a specific version of our 🤗 Datasets library for reproducibility, you can use the corresponding version Zenodo DOI from this [list](https://zenodo.org/search?q=conceptrecid:%224817768%22&sort=-version&all_versions=True).
```

**Review by albertvillanova — APPROVED** [review]
It is much clearer now when to use either the paper or the version specific Zenodo DOI. Thanks.

**Pull request merged**